fighthealthinsurance/static/js/entity_fetcher.ts

```
@@ -1,71 +1,85 @@
-	processResponseChunk,
-	done,
-	0,
-	retries);
+  return connectWebSocket(
+    backend_url,
+    data,
+    processResponseChunk,
+    done,
+    0,
+    retries,
+  );
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)